Tasks/AzurePowerShellV4/AzurePowerShell.ps1

```diff
@@ -261,6 +262,6 @@ finally {
 
     Import-Module $PSScriptRoot\ps_modules\VstsAzureHelpers_
     Remove-EndpointSecrets
-    Disconnect-AzureAndClearContext -ErrorAction SilentlyContinue
+    Disconnect-AzureAndClearContext -restrictContext $__vsts_input_restrictContextToCurrentTask -ErrorAction SilentlyContinue
```

Comment:
restrictContext has a type 'string' in the function definition, but you are reading and passing in a bool. Change the function definition to take in a boolean value

Reply:
Done

Reply:
After doing this change : 
![image](https://user-images.githubusercontent.com/45754171/95832566-107f6680-0d58-11eb-8603-7bfebecef62d.png)

I am getting this error :https://dev.azure.com/arjgupta22/p1/_releaseProgress?_a=release-environment-logs&releaseId=101&environmentId=101
That's why I kept it as string